app/controllers/PayeControllerHistoric.scala

```diff
@@ -69,14 +69,24 @@ class PayeControllerHistoric @Inject()(
         hasTaxCodeRecordsInYearPerEmployment <- hasTaxCodeRecordsFuture
       } yield {
         implicit val user = request.taiUser
-        Ok(
-          views.html.paye.historicPayAsYouEarn(
-            HistoricPayAsYouEarnViewModel(taxYear, employments, hasTaxCodeRecordsInYearPerEmployment),
-            numberOfPreviousYearsToShow))
+        isRtiUnavailable(employments) match {
+          case true => BadGateway(views.html.serviceUnavailable())
+          case false =>
+            Ok(
+              views.html.paye.historicPayAsYouEarn(
+                HistoricPayAsYouEarnViewModel(taxYear, employments, hasTaxCodeRecordsInYearPerEmployment),
+                numberOfPreviousYearsToShow))
+        }
       }
     }
   } recoverWith hodStatusRedirect
 
+  private def isRtiUnavailable(employments: Seq[Employment]): Boolean =
+    if (employments.head.annualAccounts.head.realTimeStatus.toString == "TemporarilyUnavailable")
```

Comment:
You dont need an if else to return a boolean. The statement inside the if returns a boolean. This is essentially writing ```if(true) true ese false```